howto/vscode-debugging-daprd/README.md

```diff
@@ -0,0 +1,150 @@
+
+```json
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/Game/Services/LeaderboardService/LeaderboardService.csproj",
+                "/property:GenerateFullPaths=true",
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "daprd-web",
+            "command": "daprd",
+            "args": [
+                "-dapr-id",
+                "whac-a-mole--web",
+                "-app-port",
+                "3000",
+                "-dapr-http-port",
+                "51000",
+                "-dapr-grpc-port",
+                "52000",
+                "-placement-address",
+                "localhost:50005"
+            ],
+            "isBackground": true,
+            "problemMatcher": {
+                "pattern": [
+                    {
+                      "regexp": ".",
+                      "file": 1,
+                      "location": 2,
+                      "message": 3
+                    }
+                ],
+                "background": {
+                    "beginsPattern": "^.*starting Dapr Runtime.*",
+                    "endsPattern": "^.*waiting on port.*"
+                }
+            }
+        },
+        {
+            "label": "daprd-leaderboard",
+            "command": "daprd",
+            "args": [
+                "-dapr-id",
+                "whac-a-mole--leaderboard",
+                "-app-port",
+                "5000",
+                "-dapr-http-port",
+                "51001",
+                "-dapr-grpc-port",
+                "52001",
+                "-placement-address",
+                "localhost:50005"
+            ],
+            "isBackground": true,
+            "problemMatcher": {
+                "pattern": [
+                    {
+                      "regexp": ".",
+                      "file": 1,
+                      "location": 2,
+                      "message": 3
+                    }
+                ],
+                "background": {
+                    "beginsPattern": "^.*starting Dapr Runtime.*",
+                    "endsPattern": "^.*waiting on port.*"
+                }
+            },
+            "dependsOn": "build"
+        }
+    ]
+}
+```
+
+## Wrapping up
+
+Once you have made the required changes, you should be able to switch to the debug view in VSCode and launch your "daprized" configurations by clicking the "play" button. If everything was configured correctly, you should see daprd launch in the VSCode terminal window and the debugger should attach to your application (you should see it's output in the debug window). Happy debugging!
```

Comment:
Add link to VSCode debugging https://code.visualstudio.com/Docs/editor/debugging. 

Add note that since this application was not launch using dpr "run" CLI command, but by running daprd, it, the dapr "list" does not show a list of apps that are currently running. We may have to fix this to be a better experience. 

It would be good to also add from here https://github.com/dapr/dotnet-sdk/issues/141 that you can also just add a dapr launch command that just runs the dapr CLI as a convenience to launch using the dapr CLI rather than having to type the command in each time.